src/open_inwoner/userfeed/management/commands/process_feed_updates.py

```diff
@@ -24,6 +24,6 @@ def auto_expire_items():
         completed_at__isnull=True,
     )
     for data in qs:
-        logger.info(f"automatically expired feed item: {data.id} {data}")
+        logger.info("automatically expired feed item:", id=data.id, data=data)
```

Comment:
Might as well be defensive here too:

```suggestion
        logger.info("automatically expired feed item:", id=data.id, data=str(data))
```